test/integration/app/prompted_nobuild.js

```diff
@@ -26,6 +26,7 @@ var rimraf = require('rimraf');
 var fs = require('fs');
 
 var appGeneratorPath = path.join(__dirname, '../../../app');
+var extendedTimeout = 300000
```

Comment:
Seems like a rather large timeout for nobuild tests--have you been seeing issues with timeouts?